features/step_definitions/journey/registration_steps.rb

```diff
@@ -31,11 +38,20 @@
     select_random_upper_tier_options("existing")
   end
 
-  @business_name = "Lower tier #{@organisation_type} new registration"
+  @journey.carrier_type_page.submit(choice: :carrier_broker_dealer) if @tier == "upper"
```

Comment:
Might be useful to set `@carrier = "carrier_broker_dealer"` here, but if the tests pass it's not essential.

Reply:
Nothing in the new registration flow currently uses I, but it will do no harm :) 
```suggestion
  if @tier == "upper"
    @journey.carrier_type_page.submit(choice: :carrier_broker_dealer)
    @carrier = "carrier_broker_dealer"
  end
```